go/pkg/apis/enricher/framework/java/jboss_eap_detector.go

```diff
@@ -31,5 +33,23 @@ func (o JBossEAPDetector) DoFrameworkDetection(language *model.Language, config
 }
 
 func (o JBossEAPDetector) DoPortsDetection(component *model.Component, ctx *context.Context) {
-	// Not implemented
+	var ports []int
+	jbossCli := filepath.Join(component.Path, "src/main/resources/bin/jboss-cli.xml")
```

Comment:
I'm not familiar with JBoss at all so just curious if this is where the `jboss-cli.xml` will always be stored?

Reply:
>Certain aspects of the management CLI can be customized in its configuration file, jboss-cli.xml. This file must be located either in the EAP_HOME/bin directory or in a custom directory specified with the jboss.cli.config system property.

I've found this in the [documentation for Jboss configuration](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.1/html/management_cli_guide/config_management_cli)